### PR TITLE
Add GenieGO from DirecTV

### PR DESCRIPTION
**Pull request by mcowger**

### CODE REVIEW

**Comment by vitorgalvao** [comment]
The order and separation of stanzas should be:

```
cask :v1 => 'geniego' do
  version :latest
  sha256 :no_check

  url 'http://directv.vo.llnwd.net/e4/nomad/MACOS-client.dmg'
  name 'DIRECTV GenieGO'
  name 'GenieGO'
  name 'Nomad'
  homepage 'http://www.directv.com/technology/geniego'
  license :closed

  app 'Nomad.app', :target => 'GenieGO.app'
end
```

File: Casks/geniego.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  sha256 :no_check
+
+  url 'http://directv.vo.llnwd.net/e4/nomad/MACOS-client.dmg'
+  name 'DIRECTV GenieGO'
+  name 'GenieGO'
+  name 'Nomad'
+
+  homepage 'http://www.directv.com/technology/geniego'
+
+  app 'Nomad.app', :target => 'GenieGO.app'
```
Comment (vitorgalvao) [inline Casks/geniego.rb]: Why is this being renamed? The reason for renaming needs to be documented in a comment. See the examples [linked in the documentation](https://github.com/caskroom/homebrew-cask/blob/master/doc/CASK_LANGUAGE_REFERENCE.md#target-should-only-be-used-in-select-cases).

**Comment by mcowger** [comment]
Thanks for the clarifications - I've updated the stanza ordering and added a comment about why I'm using the `target` option.

File: Casks/geniego.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+  url 'http://directv.vo.llnwd.net/e4/nomad/MACOS-client.dmg'
+  name 'DIRECTV GenieGO'
+  name 'GenieGO'
+  name 'Nomad'
+  homepage 'http://www.directv.com/technology/geniego'
+  license :closed
+
+  app 'Nomad.app', :target => 'GenieGO.app'
+  #This app is renamed because although the filename is Nomas (a reference to the old version of the marketing),
```
Comment (vitorgalvao) [inline Casks/geniego.rb]: No need for such a verbose comment. In fact, you should follow [the example](https://github.com/caskroom/homebrew-cask/blob/master/Casks/imagemin.rb) verbatim.

**Comment by mcowger** [comment]
Corrected.

**Comment by vitorgalvao** [comment]
Looks good. Could you please [squash your commits](http://davidwalsh.name/squash-commits-git)?

**Comment by mcowger** [comment]
Not worth 4 go rounds for me.

**Comment by vitorgalvao** [comment]
Very well. Thank you for the contribution. It will be merged in [a different pull request](https://github.com/caskroom/homebrew-cask/pull/10338) with the squashed commits. Your contribution is still included (and still credited to you).

**Pull request closed without merging**